content/02-new-to-cardano/10-what-is-a-smart-contract.mdx

```diff
@@ -3,24 +3,14 @@ title: What is a smart contract?
 metaTitle: What is a smart contract?
 ---
 
-A _smart contract_ is an automated digital agreement, written in code, that
-tracks, verifies, and executes the binding transactions of a contract between
-various parties. The transactions of the contract are automatically executed by
-the smart contract code when predetermined conditions are met. Essentially, a
-smart contract is a short program whose inputs and outputs are transactions on a
-blockchain.
+Smart contracts on Cardano are simple programs often referred to as validator scripts in which users define custom logic. They are automatically executed by each Cardano node validating the transaction when the transaction attempts to move funds from the script's address. The address of each script is derived from the hash-sum of the compiled script. 
 
-Smart contracts are self executing and reliable and do not require the actions
-or presence of third parties. The smart contract code is stored on, and
-distributed across, a decentralised blockchain network, making it transparent
-and irreversible.
+Once deployed, smart contracts become immutable, which means they cannot be changed. They are distributable and tamper-proof, fast, and cost-effective, as there is no middleman, which saves both money and time.
```

Comment:
Instead of "middleman," I suggest we use "intermediary" as a gender-neutral term.